Dockerfile

```diff
@@ -1,22 +1,19 @@
-# ubuntu 16.04 xenial image
-FROM ubuntu
+# use base python
+FROM python:2.7
 
-# apt update it
-RUN apt-get update
-# get essentials
-RUN apt-get install -y tar git curl wget dialog net-tools build-essential python python-dev python-distribute python-pip
+# create app directory
+RUN mkdir -p /app/
+WORKDIR /app/
 
-# add project directory
-ADD . /telemetry-schema-service/
+# install requirements
+COPY requirements.txt /app/
+RUN pip install --no-cache-dir --requirement requirements.txt
 
-# pip install requirements.txt
-RUN pip install -r /telemetry-schema-service/requirements.txt
-
-# exposing 8080 for flask app
+# expose 8080 for flask app
 EXPOSE 8080
 
-# workdir
-WORKDIR /telemetry-schema-service/app
+# copy application
+COPY . /app/
```

Comment:
Meta point, maybe we should generate the requirements.txt with `pip freeze` so it has version info? Doesn't have to be part of this PR though.